recirq/qml_lfe/learn_dynamics_c_test.py

```diff
@@ -15,13 +15,12 @@
 import os
 import pytest
 import numpy as np
-from . import learn_dynamics_c
 
 
 def test_shadows_dont_seperate(tmpdir):
     """Ensure that classical shadows cannot distinguish tsym vs scrambling circuits."""
+    from . import learn_dynamics_c
```

Comment:
Why move this down?  Generally, I think the imports should go at the top unless there is a good reason for moving it into the function (such as circular imports).  For instance, if we added a second test, we'd have to move it back up anyway.

Reply:
That was originally a workaround to the fact that FLAGS are global and meant to work with bazel (where tests are made into binaries) vs. pytest where things are just imported. I have a cleaner fix now.